src/support/file.h

```diff
@@ -29,15 +29,15 @@
 
 namespace wasm {
 template <typename T>
-T read_file(const std::string &filename, bool debug);
+T read_file(const std::string &filename, bool binary, bool debug);
```

Comment:
It's a pretty common technique in C/C++ API design. It makes it much easier to understand the call site:
```
read_file("foo.bar", false, true); // what does this mean?
```
vs.
```
read_file("foo.bar", kAsText, kDebug);
```